src/txmempool.h

```diff
@@ -237,6 +239,21 @@ class CompareTxMemPoolEntryByEntryTime
 
 class CBlockPolicyEstimator;
 
+/**
+ * Information about a mempool transaction.
+ */
+struct TxMempoolInfo
```

Comment:
(bbfd39da096e7c05c93f409869b4c75afb454f66)
This is never used right now. Guess it will come later, when bitcoin/bitcoin#8126 is fully backported  (maybe we it should  be added later, then, for ease of review).

Reply:
Yes, done.